website/docs/installation/documentdb/deb.md

```diff
@@ -4,6 +4,15 @@ sidebar_position: 2
 
 # DEB package
 
+:::note
+We provide different `.deb` packages for various deployments.
+
+- For most use cases, we recommend using the production package (e.g., `documentdb.deb`).
+- For debugging purposes, use the development package (contains either `-dev` or `-dbgsym` suffix e.g., `documentdb-dev.deb`/`documentdb-dbgsym.deb`).
+  It includes features that significantly slow performance and is not recommended for production use.
+
+:::
```

Comment:
I don't understand why it is bolted on top of the page instead of being integrated. We should continuously update our documentation, not just add on top of it, making _the same page_ say the thing multiple times.